packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/go/emailcallbackuri.md

```diff
@@ -0,0 +1,3 @@
+8. In the **General Settings** section, click **Edit**.
+9. Under **EMAIL VERIFICATION EXPERIENCE** set **Callback URI** to a URI that is appropriate for your app. For example, `http://localhost:8000` if you're using the sample app.
```

Comment:
The "`http://localhost:8000`" value the callback URI, doesn't look right to me. Can you confirm with Mike Mondragon this is correct?

Reply:
I think you're right. It looks like localhost:8000/login on further inspection. Have slacked Mike to confirm.